app/assets/javascripts/admin/orders/controllers/bulk_invoice_controller.js.coffee

```diff
@@ -21,7 +21,7 @@ angular.module("admin.orders").controller "bulkInvoiceCtrl", ($scope, $http, $ti
     .error (data) ->
       $scope.poll++
 
-      if $scope.poll > 10
+      if $scope.poll > 30
```

Comment:
:warning: magic number. We should assign it to a constant with meaningul name and doc. Can be done in a separate pr if you prefer